tests/unit/verticals/secret/output/snapshots/snap_test_text_output.py

```diff
@@ -258,15 +245,14 @@
    Incident URL: N/A
    Secret SHA: 060bf63de122848f5efa122fe6cea504aae3b24cea393d887fdefa1529c6a02e
 
-    | @@ -0,0 +1,29 @
   1 | …**5733 /ce3f9f********************5e43711 -----BEGIN RSA PRIVATE KEY-----
-  2 | +MIIBOgIBAAJBAIIRkYjxjE3KIZi******************************+******
-  3 | +****************************************************************
-  4 | +****************************************************************
-  5 | +***********+****************************************************
-  6 | +****************+***********************************************
-  7 | +**********************+*****************************************
-  8 | +****+******Xme/ovcDeM1+3W/UmSHYUW4b3WYq4
+  2 | MIIBOgIBAAJBAIIRkYjxjE3KIZi******************************+******
+  3 | ****************************************************************
+  4 | ****************************************************************
+  5 | ***********+****************************************************
+  6 | ****************+***********************************************
+  7 | **********************+*****************************************
+  8 | ****+******Xme/ovcDeM1+3W/UmSHYUW4b3WYq4
```

Comment:
I think it's not a good idea to loose the "category" character: it is useful for users to know if the scanned commit added or removed the secret.